akka-actor/src/main/scala/akka/event/EventStream.scala

```diff
@@ -34,9 +34,9 @@ class EventStream(sys: ActorSystem, private val debug: Boolean) extends LoggingB
     def isSubclass(x: Class[_], y: Class[_]) = y isAssignableFrom x
   }
 
-  protected def classify(event: AnyRef): Class[_] = event.getClass
+  protected def classify(event: Any): Class[_] = event.getClass
```

Comment:
`javap` says that this is the same both before and after the change, which is good

```
public java.lang.Class<?> classify(java.lang.Object);
```